src/Sylius/Bundle/ContentBundle/Resources/translations/messages.th.yml

```diff
@@ -88,9 +88,9 @@ sylius:
             title: หัวข้อ
         static_content:
             body: Body
-            id: ไอดี
+            id: ID
             internal_name: ชื่อภายใน
-            menu_nodes: เมนูต่างๆ
+            menu_nodes: Menu nodes
```

Comment:
It is correct in terms of SyliusBot work, the Thai translation was changed on CrowdIn. Since we don't verify translations, there's no guarantee that translations make sense.